src/index.js

```diff
@@ -24,27 +24,24 @@ program
   .action(async (outputLang, inputFiles) => {
     const outputFile = program.opts().output;
 
-		// Loop through file path args
+    // Loop through file path args
     for (let filePath of inputFiles) {
       try {
         const fileContent = await fs.readFile(filePath, { encoding: "utf8" });
-        const responseStream = await getGroqChatStream(
-          fileContent,
-          outputLang
-        );
-				// Check if --output flag was used
+        const responseStream = await getGroqChatStream(fileContent, outputLang);
+        // Check if --output flag was used
         if (outputFile) {
           let response = "";
-					// Read response stream one chunk at a time
-					// Store chunks in `response` for writing to output file
+          // Read response stream one chunk at a time
+          // Store chunks in `response` for writing to output file
           for await (const chunk of responseStream) {
             const chunkContent = chunk.choices[0]?.delta?.content || "";
-            process.stdout.write(chunkContent);
             response += chunkContent;
           }
           await fs.writeFile(outputFile, `${response}\n`);
+          process.stdout.write(`Output written to ${outputFile}`);
```

Comment:
Prefer silence on success when `--output` is used, in line with the philosophy of tools like `grep`, `sed`, and `awk`.